pages/common/curl.md

```diff
@@ -5,32 +5,40 @@
 
 - Download a URL to a file:
```

Comment:
I agree @agnivade's proposal is clearer. I vote for updating wget.md accordingly.